### ISSUE TITLE
Improvements for documentation

### ISSUE THREAD

**Issue by bbockelm** [jose-caballero/cvmfsreplica#3]
There are some improvements needed in the documentation (https://twiki.grid.iu.edu/bin/view/Documentation/Release3/CvmfsReplica):

* It should cover an end-to-end example of replicating a repo, as in the existing CVMFS Stratum-1 documentation.
* Should refer to OSG yum repos, not direct RPM download.
* Should have examples of systemd-based services.
* What needs to be done to add Stratum-1s with public keys like OASIS?

Basically, I'd like to be able to follow the directions and end up with a Stratum-1 for OASIS.

**Comment by jose-caballero** [comment]
Hi,

do you want all details to be included in this twiki?
I added a section https://twiki.grid.iu.edu/bin/view/Documentation/Release3/CvmfsReplica#7_Configuration  that points to other pieces of documentation. Would that work?
Basically, I am afraid of the need of updating two twikies when something changes on the pure CVMFS part of the business. 

I will fix/add the other items. 

Jose

**Comment by bbockelm** [comment]
Hi Jose,

I was thinking more along the lines of creating a complete copy of https://twiki.grid.iu.edu/bin/view/Documentation/Release3/InstallCvmfsStratum1, then embedding the `cvmfsreplica` documentation there.

The idea is not to maintain two permanent forks but rather have the copy replace the original.  Hence, no need to worry about drift of documentation.

Brian
